examples/tutorials/first-figure.py

```diff
@@ -34,11 +35,12 @@
 fig = pygmt.Figure()
 
 ########################################################################################
-# Add elements to the figure using its methods. For example, let's start a map with an
-# automatic frame and ticks around a given longitude and latitude bound, set the
-# projection to Mercator (``M``), and the map width to 8 inches:
+# Add elements to the figure using its methods. For example, let's use
+# :meth:`pygmt.Figure.basemap` to start a map for a region indicated by a given
+# longitude and latitude bound, set the projection to Mercator (``M``), the
+# map width to 8 centimeters, and frame type to be generated automatically:
 
-fig.basemap(region=[-90, -70, 0, 20], projection="M8i", frame=True)
+fig.basemap(region=[-90, -70, 0, 20], projection="M8c", frame=True)
```

Comment:
Is it better to use SI?

Reply:
Yes, we should use SI units.

Reply:
There's another M8i at Line 61 to change.

Reply:
> There's another M8i at Line 61 to change.

Revised in 5f67156.